forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -442,6 +447,49 @@ fn build_opts_from_cmd(cmd: &cmd::Deploy) -> pkg::BuildOpts {
     }
 }
 
+fn create_deployment_artifact(
```

Comment:
Would you mind adding a doc comment here outlining the purpose and intended use of this function?

Reply:
done ✅